code-challenges.test.js

```diff
@@ -38,8 +65,32 @@ const votes1 = { upVotes: 13, downVotes: 2 }
 const votes2 = { upVotes: 2, downVotes: 33 }
 // Expected output: -31
 
+// JEST Test
+// describe finalTally
+// it takes in the two values of votes1, determines which is the larger value, and performs subtraction to get the finalTally
+// expected output: 11
+// it takes in the values of votes 2, compares the two values, and perform addition to get the finalTally
+// expected output: -31
+
 // b) Create the function that makes the test pass.
 
+// Psuedo-code
+// 1. Create function finalTally
+// 2. finalTally takes the two values of each constant
+// 3. use comparison operator to determine which value is larger
+// 4. use addition to obtain finalTally of each constant provided
+// 5. Consol.log the results for both of the constants
+
+const finalTally = (upVotes, downVotes) => {
+  if (upVotes > downVotes) {
+    return(upVotes - downVotes)
+    } else {return(-(downVotes) + upVotes)
+    }
+  }
+
+  console.log(finalTally(13, 2))
+  console.log(finalTally(2, 33))
```

Comment:
After you have researched dot notation, please try to rework this prompt.
Need to see your test and function.